dts/bindings/led/ti,lp5569.yaml

```diff
@@ -3,3 +3,16 @@ description: TI LP5569 LED
 compatible: "ti,lp5569"
 
 include: i2c-device.yaml
+
+properties:
+  charge-pump-mode:
+    type: string
+    default: "disabled"
+    enum:
+      - "disabled"
+      - "1x"
+      - "1x5"
+      - "auto"
+    description: |
```

Comment:
@simonguinot Shouldn't a default value be noted in the description?

Reply:
Well, it is the rule according to https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#rules-for-default-values. So yes :)

Reply:
Ah, right. Sorry about that! Should be done now